PYME/DSView/eventLogViewer.py

```diff
@@ -32,6 +32,16 @@
 from matplotlib import cm
 from PYME.Analysis.piecewiseMapping import times_to_frames, frames_to_times
 
+# python 3.10 compat definitions
```

Comment:
I sort of understand why you've done this (to avoid boilerplate changes), but think it would be preferable in terms of long-term readability and maintainability if the `int()` casts were made in functions themselves. This is probably fixable with a regex search and replace ...